app/scripts/views/sign_up.js

```diff
@@ -183,10 +183,18 @@ define(function (require, exports, module) {
     beforeDestroy () {
       var formPrefill = this._formPrefill;
       formPrefill.set('email', this.getElementValue('.email'));
-      formPrefill.set('password', this.getElementValue('.password'));
+
+      const password = this.getElementValue('#password');
+      if (password === this.getElementValue('#password-confirm')) {
```

Comment:
I'd just save this regardless. Can you use the id `#vpassword` to be consistent with [the complete_reset_password view](https://github.com/mozilla/fxa-content-server/blob/d136eecba1abe2864bb81ef5d7fc9c1ab49e1ffb/app/scripts/views/complete_reset_password.js#L147)?